src/components/info-card/ambassador-data.tsx

```diff
@@ -0,0 +1,470 @@
+    organization: "GraphQL.js",
+    tags: [
+      {
+        label: "GitHub",
+        url: "https://github.com/jovidecroock",
+        icon: <GitHubIcon className="h-5 w-5" />,
+      },
+      {
+        label: "LinkedIn",
+        url: "https://www.linkedin.com/in/jovidecroock/",
+        icon: <LinkedInIcon className="h-5 w-5" />,
+      },
+      {
+        label: "X",
+        url: "https://x.com/jovidecroock",
+        icon: <TwitterIcon className="h-5 w-5" />,
+      },
+      {
+        label: "Website",
+        url: "https://www.jovidecroock.com/blog/",
+        icon: <GlobeIcon className="h-5 w-5" />,
+      },
+    ],
+  },
+  {
+    label: "Marc-Andre Giroux",
+    imageUrl: "https://github.com/xuorig.png",
+    alt: "Marc-Andre Giroux",
+    organization: "Netflix",
+    tags: [
+      {
+        label: "BlueSky",
+        url: "https://bsky.app/profile/xuorig.bsky.social",
+        icon: <Bluesky2Icon className="h-5 w-5" />,
+      },
+      {
+        label: "GitHub",
+        url: "https://github.com/xuorig",
+        icon: <GitHubIcon className="h-5 w-5" />,
+      },
+      {
+        label: "LinkedIn",
+        url: "https://www.linkedin.com/in/magiroux/",
+        icon: <LinkedInIcon className="h-5 w-5" />,
+      },
+      {
+        label: "Website",
+        url: "https://magiroux.com/",
+        icon: <GlobeIcon className="h-5 w-5" />,
+      },
+    ],
+  },
+  {
+    label: "Michael Watson",
+    imageUrl: "https://github.com/michael-watson.png",
+    alt: "Michael Watson",
+    organization: "Apollo",
+    tags: [
+      {
+        label: "BlueSky",
+        url: "https://bsky.app/profile/michael-watson.bsky.social",
+        icon: <Bluesky2Icon className="h-5 w-5" />,
+      },
+      {
+        label: "GitHub",
+        url: "https://github.com/michael-watson",
+        icon: <GitHubIcon className="h-5 w-5" />,
+      },
+      {
+        label: "LinkedIn",
+        url: "https://www.linkedin.com/in/michael-watson-%F0%9F%96%A5-85844442/",
+        icon: <LinkedInIcon className="h-5 w-5" />,
+      },
+    ],
+  },
+  {
+    label: "Patrick Arminio",
+    imageUrl: "https://github.com/patrick91.png",
+    alt: "Patrick Arminio",
+    organization: "Strawberry GraphQL",
+    tags: [
+      {
+        label: "BlueSky",
+        url: "https://bsky.app/profile/patrick.wtf",
+        icon: <Bluesky2Icon className="h-5 w-5" />,
+      },
+      {
+        label: "GitHub",
+        url: "https://github.com/patrick91",
+        icon: <GitHubIcon className="h-5 w-5" />,
+      },
+      {
+        label: "LinkedIn",
+        url: "https://www.linkedin.com/in/patrickarminio/",
+        icon: <LinkedInIcon className="h-5 w-5" />,
+      },
+    ],
+  },
+  {
+    label: "Phil Pluckthun",
+    imageUrl: "https://github.com/kitten.png",
+    alt: "Phil Pluckthun",
+    organization: "urql & gql.tada",
+    tags: [
+      {
+        label: "BlueSky",
+        url: "https://bsky.app/profile/patrick.wtf",
```

Comment:
Cought this on a quick read though. Thanks for getting this page put together and published 🙌 

```suggestion
        url: "https://bsky.app/profile/kitten.sh",
```

Reply:
Oh no! Thanks for checking!